src/pages/markets/static/content/_basket.tsx

```diff
@@ -2,6 +2,7 @@ import { TMarket, TSimpleStepContent } from './_types'
 import Baskets from 'images/svg/stock-indices/baskets.svg'
 import AdvancedCharting from 'images/svg/markets/advanced-charting-widgets-new.svg'
 import Leverage from 'images/svg/stock-indices/stocks-high-leverage.svg'
+import TightSpread from 'images/svg/markets/tight-spread-new.svg'
```

Comment:
is this icon being used?

Reply:
It's being used in the basket and the cryptocurrencies icons